bin/ruby-build

```diff
@@ -244,24 +244,39 @@ compute_md5() {
   fi
 }
 
+has_checksum_support() {
+  local checksum_command="$1"
+  local has_checksum_var="HAS_CHECKSUM_SUPPORT_${checksum_command}"
+
+  if [ -z "${!has_checksum_var+defined}" ]; then
+    printf -v "${has_checksum_var}" '%s' $(echo test | "$checksum_command" >/dev/null; echo $?)
```

Comment:
Since we're certain that the subexpression will always output a single integer, let's use it as a format string directly instead of `%s`. Also, double-quote it, even if it's not necessary, for consistency.